README.md

```diff
@@ -43,10 +44,10 @@ Or play with [this starter in your browser][stackblitz] on StackBlitz.
 
 Latest version available for each version of Angular
 
-| ng2-dragula | Angular    |
-| ---------- | ----------- |
-| 2.1.1      | <= 9.x      |
-| current    | >= 10.x     |
+| ng2-dragula | Angular |
+| ---------- |---------|
+| 2.1.1      | <= 9.x  |
+| current    | ^14.x.x |
```

Comment:
It seems that it is being built this way, but is there a reason it's restricted to `^14.x.x`? I'm using a project that just upgraded to Angular 15 and now `ng2-dragula` is failing.

Reply:
yeah, our library hasn't been updated yet, so now it fails. It will be updated soon